tools/client_package_initializer.py

```diff
@@ -105,8 +113,23 @@ def _load_setup(self) -> None:
 
         self._setup_data.update(setup)
 
+    @classmethod
+    def _append_missing_requirements(cls, setup: Dict[str, List]):
+        # for assisted_service_api.py
+        if "kubernetes" not in "\n".join(setup[Setup.INSTALL_REQUIRES_KEY]):
+            setup[Setup.INSTALL_REQUIRES_KEY].append("kubernetes~=18.20.0")
```

Comment:
please change to a broader range when listing dependencies in ``install_requires``